### PR TITLE
Typo "Teams Room"→"Teams Rooms"

### PR DESCRIPTION
**Pull request by hyoshioka0128**
https://learn.microsoft.com/en-us/microsoftteams/meeting-policies-audio-and-video 
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/public/Teams/meeting-policies-audio-and-video.md 
#PingMSFTDocs

### CODE REVIEW

File: Teams/meeting-policies-audio-and-video.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ To specify who can use far end camera control in a meeting, create and assign a
 |`AutoAcceptAll`|PTZ camera controls are automatically available to any meeting participant.|
 |`AutoAcceptInTenant`|PTZ camera controls are automatically available only to participants in the same organization as the Teams Room.|
 
-When `TeamsCameraFarEndPTZMode` is set to `AutoAcceptAll` or `AutoAcceptInTenant`, camera control can still be manually turned off from the Teams Room at any point during a meeting. Camera control is also unavailable when the camera is turned off.
+When `TeamsCameraFarEndPTZMode` is set to `AutoAcceptAll` or `AutoAcceptInTenant`, camera control can still be manually turned off from the Teams Rooms at any point during a meeting. Camera control is also unavailable when the camera is turned off.
```
Comment (JohanFreelancer9) [inline Teams/meeting-policies-audio-and-video.md:178]:
```suggestion
When `TeamsCameraFarEndPTZMode` is set to `AutoAcceptAll` or `AutoAcceptInTenant`, camera control can still be manually turned off from Teams Rooms at any point during a meeting. Camera control is also unavailable when the camera is turned off.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @hyoshioka0128, kindly commit the suggested change. Many thanks.

**Comment by hyoshioka0128** [comment]
@JohanFreelancer9 - Thanks for the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**